docs/environments/gke.md

```diff
@@ -63,3 +62,74 @@ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=clu
 
 Once all pods are running you can return to the [User Guide](../userguide.md) to deploy the
 KubeFed control-plane.
+
+## Additional firewall rule for kubefed-admission-webhook
+
+Admission control in GKE requires an additional VPC network firewall rule.
+
+### Background
+
+The GCP Kubernetes control plane is deployed in a Google-managed account.
+
+During cluster provisioning, GKE automatically creates:
```

Comment:
Are there any links to the GKE docs describing this? If so, please add references.